packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -624,4 +625,436 @@ describe('ListView', function () {
       expect(grid.scrollTop).toBe(0);
     });
   });
+
+  describe('drag and drop', function () {
+    function DraggableListView(props) {
+      let {dragHookOptions, listViewProps} = props;
+      return (
+        <>
+          <Droppable onDrop={onDrop} />
+          <DragExample dragHookOptions={{onDragStart, onDragMove, onDragEnd, ...dragHookOptions}} listViewProps={{onSelectionChange, ...listViewProps}} />
+        </>
+      );
+    }
+    beforeEach(() => {
+      jest.spyOn(HTMLElement.prototype, 'getBoundingClientRect').mockImplementation(() => ({
+        left: 0,
+        top: 0,
+        x: 0,
+        y: 0,
+        width: 100,
+        height: 50
+      }));
+    });
+
+    afterEach(() => {
+      act(() => {jest.runAllTimers();});
+      jest.clearAllMocks();
+    });
+
+    describe('via mouse', function () {
+      it('should show a default drag preview on drag', function () {
```

Comment:
From my understanding, pointer events don't actually do anything here, the important bit is that a drag event is fired. When using keyboard, we don't need to fire this drag event ourselves, it is fired by this [onPress handler](https://github.com/adobe/react-spectrum/blob/ListView-DnD/packages/@react-aria/dnd/src/useDrag.ts#L165-L194)

Reply:
yeah, but it's important to capture events a user would create, for instance, if you did this on something that had pointerDown listeners in a parent, you'd want to know if that event bubbles up and causes problems
It's the same reason I frequently comment saying that all keyDowns must be followed by their keyUp

Reply:
fair point, I'll go ahead and add those

Reply:
👍🏻 obviously we'll be within reason, it'd be unreasonable to track all events everywhere all the time
This one just seems more important specifically because you ran into this issue earlier if I recall?

Reply:
I actually don't recall haha, but yeah the pointer events def reasonable enough to include here